pwa/app/lib/eventUtils.ts

```diff
@@ -88,3 +90,31 @@ export function getEventWeekString(event: Event) {
       return `Week ${event.week + 1}`;
   }
 }
+
+export function getCurrentWeekEvents(events: Event[]) {
+  const now = new Date();
+  //const now = new Date('Jul 17, 2024');
+  const filteredEvents = [];
+
+  const diffFromWeekStart = now.getDay();
+  const closestStartMonday = new Date(now).setDate(
+    now.getDate() - diffFromWeekStart,
+  );
+
+  for (const event of events) {
+    const startDate = parseDate(event.start_date);
+
+    const timeOffsetDays = Math.floor(
+      convertMsToDays(startDate - closestStartMonday),
+    );
+
+    if (timeOffsetDays === 0 || (timeOffsetDays > 0 && timeOffsetDays < 7)) {
```

Comment:
Not sure why the check for `=== 0` and `> 0` are separate, and not just `>=`. I know the python code does this but I'm not sure why that does it either.